packages/sources/peregrine-fund-admin/test/integration/fixtures.ts

```diff
@@ -0,0 +1,60 @@
+import nock from 'nock'
+
+export const mockNAVResponseSuccess = (): nock.Scope =>
+  nock('https://fund-admin-data-adapter-v1-960005989691.europe-west2.run.app', {
+    encodedQueryParams: true,
+  })
+    .get('/api/v1/nav/')
+    .query({
+      assetId: 100,
```

Comment:
I think you'll need to fix this once you add assetId to the path rather than as a query param

Reply:
Thank you, removed. I have not worked with nock before, therefore claim no prior knowlege